.github/workflows/ci.yml

```
@@ -15,14 +15,15 @@ jobs:
         with:
           shell-action: carton bundle --product TokamakDemo
 
-  swiftwasm_test_5_6:
-    runs-on: ubuntu-20.04
+  # FIXME: re-enable after https://github.com/swiftwasm/WasmTransformer/issues/18 is fixed.
+  # swiftwasm_test_5_6:
+  #   runs-on: ubuntu-20.04
 
-    steps:
-      - uses: actions/checkout@v2
-      - uses: swiftwasm/swiftwasm-action@v5.6
-        with:
-          shell-action: carton test
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: swiftwasm/swiftwasm-action@v5.6
+  #       with:
+  #         shell-action: carton test
```

I wonder if this will work now with new `carton test --environment node` introduced in `carton` 0.15? I've updated the GitHub Action to that latest version already.

```suggestion
    steps:
      - uses: actions/checkout@v2
      - uses: swiftwasm/swiftwasm-action@v5.6
        with:
          shell-action: carton test --environment node
```

---

Yeah, I've verified that `carton test --environment node` works locally on macOS, so I think we should re-enable the tests this way.